ClosedXML_Tests/TestHelper.cs

```diff
@@ -91,12 +96,19 @@ public static void RunTestExample<T>(string filePartName, bool evaluateFormulae
                 using (var streamActual = File.OpenRead(filePath2))
                 {
                     var success = ExcelDocsComparer.Compare(streamActual, streamExpected, out string message);
-                    var formattedMessage =
-                        String.Format(
-                            "Actual file '{0}' is different than the expected file '{1}'. The difference is: '{2}'",
-                            filePath2, resourcePath, message);
+                    var formattedMessage = $"Actual file is different than the expected file '{resourcePath}'. The difference is: '{message}'.";
 
-                    Assert.IsTrue(success, formattedMessage);
+                    if (success)
+                    {
+                        return;
+                    }
+
+                    if (string.IsNullOrEmpty(allowedDiff))
+                    {
+                        Assert.Fail(formattedMessage);
+                    }
+
+                    Assert.That(message, Is.EqualTo(allowedDiff));
```

Comment:
there could be a better message stating that there is an allowedDiff using the form `Assert.That(message, Is.EqualTo(allowedDiff), "allowed diff bla bla");`

Reply:
$"Actual diff '{message}' differs to expected diff '{expectedDiff}', file '{resourcePath}'"